apps/src/code-studio/pd/workshop_enrollment/enroll_form.jsx

```diff
@@ -93,6 +99,8 @@ const REPLACE_EXISTING_OPTIONS = [
   'I don’t know'
 ];
 
+const ERROR_COLOR = '#a94442'; // Error color used in 'react-bootstrap'
```

Comment:
please extract color constants to colors.scss, or use one that's there already. I would suggest using this one since it is quite similar: https://github.com/code-dot-org/code-dot-org/blob/abc30834e8a18eb81d0cd7dc2e7f6d425e208b7a/shared/css/color.scss#L112 

if that looks wrong to you, perhaps you can figure out why they are different (different bootstrap versions?) and add a new constant with an disambiguating name?

Reply:
Good catch! I [posted in the #design channel](https://codedotorg.slack.com/archives/C0SUN2SSF/p1675955877959669) about this –– short version is that it looks like these constants are coming from here https://bootstrapdocs.com/v2.0.0/docs/less.html, but the actual bootstrap we use come from https://getbootstrap.com/docs/3.4/customize/. Unsure if I should make new variables for this, or replace existing ones.

Reply:
How about adding new variables and including "v3" in the name?

Reply:
Yeah, that's easier than changing existing constants –– sounds good!